scripts/plotting/tem.py

```diff
@@ -74,11 +72,15 @@ def tem(adf):
     print(f"\t NOTE: redo_plot is set to {redo_plot}")
     #-----------------------------------------
 
-    #Location to saved TEM netCDF files
-    tem_loc = adf.get_basic_info("tem_loc")
+    #Grab TEM diagnostics options
+    tem_opts = adf.read_config_var("tem_info")
+
+    #Location of saved TEM netCDF files
+    tem_loc = tem_opts["tem_loc"]
```

Comment:
I think we need to use the `get` method here:

```suggestion
    tem_loc = tem_opts.get("tem_loc")
```